### ISSUE TITLE
FeatureSet record has empty ReferenceSet field

### ISSUE THREAD

**Issue by macieksmuga** [ga4gh/server#1149]
The current implementation of `FeatureSet` is not properly initialized with a valid `ReferenceSetId`, and the corresponding field in responses to `featuresets/search` and `featuresets/<id>` is currently left blank.

**Comment by jeromekelleher** [comment]
This is resolved in the sql repo branch.

**Comment by macieksmuga** [comment]
Not sure I see how - I've been testing the `jeromekelleher/remove-filesystemdataset` branch today (which I believe is the latest SQL repo manager codebase), and `addFeatureSet` (as well as `removeFeatureSet`) is not implemented in the `RepoManager` object there yet. Unless there's some other place I should be looking, I'll supply the needed code.

**Comment by jeromekelleher** [comment]
It looks like something weird has happened with that branch @macieksmuga, sorry about that. It's turned into a bit of a mess trying to keep all these PRs going at the same time. Here's what happens with the code in the ``sql_repo`` branch now:
```
jk@holly$ python repo_dev.py init test.db  -f
jk@holly$ python repo_dev.py add-dataset test.db ds
jk@holly$ python repo_dev.py add-featureset test.db ds tests/data/datasets/dataset1/sequenceAnnotations/gencodeV21Set1.db
repo_dev.py: error: a reference set name must be provided
jk@holly$ python repo_dev.py add-featureset test.db ds tests/data/datasets/dataset1/sequenceAnnotations/gencodeV21Set1.db -R NCBI37
repo_dev.py: error: ReferenceSet with name 'NCBI37' not found
jk@holly$ python repo_dev.py add-referenceset test.db tests/data/referenceSets/NCBI37.fa.gz
jk@holly$ python repo_dev.py add-featureset test.db ds tests/data/datasets/dataset1/sequenceAnnotations/gencodeV21Set1.db -R NCBI37
```

**Comment by jeromekelleher** [comment]
Closed by #1166.
